hail/python/hail/ggplot/geoms.py

```diff
@@ -545,3 +545,145 @@ def get_stat(self):
 
 def geom_func(mapping=aes(), fun=None, color=None):
     return GeomFunction(mapping, fun=fun, color=color)
+
+
+class GeomArea(Geom):
+    aes_to_arg = {
+        "fill": ("fillcolor", "black", True),
+        "color": ("line_color", "rgba(0, 0, 0, 0)", True),
+        "tooltip": ("hovertext", None, False),
+        "fill_legend": ("name", None, True)
+    }
+
+    def __init__(self, aes, fill, color):
+        super().__init__(aes)
+        self.fill = fill
+        self.color = color
+
+    def apply_to_fig(self, parent, agg_result, fig_so_far, precomputed, num_groups):
+        def plot_group(df):
+            scatter_args = {
+                "x": df.x,
+                "y": df.y,
+                "fill": 'tozeroy'
+            }
+
+            for aes_name, (plotly_name, default, take_one) in self.aes_to_arg.items():
+                if hasattr(self, aes_name) and getattr(self, aes_name) is not None:
+                    scatter_args[plotly_name] = getattr(self, aes_name)
+                elif aes_name in df.columns:
+                    if take_one:
+                        scatter_args[plotly_name] = df[aes_name].iloc[0]
+                    else:
+                        scatter_args[plotly_name] = df[aes_name]
+                elif default is not None:
+                    scatter_args[plotly_name] = default
+
+            fig_so_far.add_scatter(**scatter_args)
+
+        groups = set(agg_result["group"])
```

Comment:
Change this to be `range(num_groups)`. I added the `num_groups` argument to make the order groups get plotted be consistent.